eva/udfs/abstract/hf_abstract_udf.py

```diff
@@ -59,6 +68,8 @@ def output_formatter(self, outputs: Any):
         """
         Function that formats output from HuggingFace format to EVA format (pandas dataframe)
         """
+        if isinstance(outputs, dict):
```

Comment:
Just for my understanding, can you give an example of this type of ouput. Is it of the type `{labels: [1,2,1,2,1], class: ['car', 'bus', 'car', 'bus', 'car']}. We probably need examples in the docstrings on what are the type of outputs that we expect here. I can do it for the others but you could mention for audio.

Reply:
For automatic speech recognition, this will be like {'text':<transcript_from_video>}>. The default code doesn't really handle it, plus this way looks cleaner. Will add an example